### DISCUSSION TITLE
Q: Can htmx 2.0 be transpiled to es5 safely?

### DISCUSSION THREAD

**Discussion by adonespitogo** [bigskysoftware/htmx#3044] [Q&A]
Sorry if this is a dumb question. I'm in a tight schedule and doesn't have the time to test if this is working.

Since I've read that htmx 2.x now supports ES6 syntax, I wonder if it uses ES6 syntax like `Proxy` the same as [alpinejs](https://github.com/alpinejs/alpine/discussions/3568), which prevent it from being used in older browsers even if alpine is transpilable to es5.

**Comment by Telroshan** [comment]
> I wonder if it uses ES6 syntax like Proxy

Yep, we're definitely using that one, see our [FormData proxy](https://github.com/bigskysoftware/htmx/blob/master/src/htmx.js#L4037)
So no, htmx will not compile _safely_ to es5, it might, or might not, maybe some tools can automatically convert it to es5 syntax, I have no idea.
But if you need this support, or need to support IE11, htmx 1 is still available & will still be maintained
